README.md

```diff
@@ -11,3 +11,21 @@ Install dependencies via
 - [Stellar](./stellar/README.md)
 - [XRPL](./xrpl/README.md)
 
+## Javascript -> Typescript Migration
+
+Please note that this project currently supports both Javascript and Typescript.
```

Comment:
Should people still use `node ...` or `ts-node ...` is preferred even if the old way is supported? Clarify the usage in the docs